src/main/java/com/gotocompany/depot/redis/client/RedisStandaloneClient.java

```diff
@@ -31,16 +31,20 @@ public class RedisStandaloneClient implements RedisClient {
     private final DefaultJedisClientConfig defaultJedisClientConfig;
     private final HostAndPort hostAndPort;
     private Jedis jedis;
+    private final int connectionMaxRetries;
+    private final long connectionRetryBackoffMs;
 
-    public RedisStandaloneClient(Instrumentation instrumentation, RedisTtl redisTTL, DefaultJedisClientConfig defaultJedisClientConfig, HostAndPort hostAndPort) {
+    public RedisStandaloneClient(Instrumentation instrumentation, RedisTtl redisTTL, DefaultJedisClientConfig defaultJedisClientConfig, HostAndPort hostAndPort, int connectionMaxRetries, long connectionRetryBackoffMs) {
```

Comment:
Let's take sinkconfig here, instead of taking so many arguments.

Reply:
fixed